project/Helpers.scala

```diff
@@ -117,7 +118,10 @@ object C {
           "com.typesafe.akka" %% "akka-testkit" % V.akka % Test,
           "ch.qos.logback" % "logback-classic" % V.logback,
           "org.scalatest" %% "scalatest" % V.scalatest % "it, test",
-          "com.dimafeng" %% "testcontainers-scala-scalatest" % "0.40.12" % "it"),
+          "com.typesafe.scala-logging" %% "scala-logging" % "3.9.5",
+          "com.dimafeng" %% "testcontainers-scala-scalatest" % "0.40.12" % "it",
+          "com.typesafe.akka" %% "akka-serialization-jackson" % "2.7.0" % Test
```

Comment:
Does 'Test' include the "it" scope as well?

Reply:
Don't think it does, will add it in then 👍

Reply:
No.